superset-frontend/src/views/datasetList/DatasetList.tsx

```diff
@@ -141,44 +146,62 @@ class DatasetList extends React.PureComponent<Props, State> {
     {
       Cell: ({
         row: {
-          original: { explore_url: exploreUrl, table_name: datasetTitle },
+          original: { kind },
         },
-      }: any) => <a href={exploreUrl}>{datasetTitle}</a>,
-      Header: t('Table'),
-      accessor: 'table_name',
+      }: any) =>
+        kind === 'physical' ? <DatasetPhysicalIcon /> : <DatasetVirtualIcon />,
```

Comment:
Will the user understand the difference between these two icons? Should we maybe add a tooltip?

Reply:
Yes, the icons are not exactly descriptive. There is the `Type` column which just renders `kind`, but a tooltip would probably go a long way here.